src/translations/nbTranslations.json

```diff
@@ -165,7 +165,7 @@
     "read_more": "Les mer",
     "read_more_about_nva": "Les mer om NVA",
     "registration": "Registrering",
-    "registrations": "Registreringer",
+    "registrations": "Resultatregistreringer",
```

Comment:
kanskje legg inn ny key `results` her? `CristinPersonList` tror jeg forøvrig skal ha table heading `Resultat` og ikke `Resultatregistreringer`